geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/api/GeoMesaFeatureIndex.scala

```diff
@@ -27,13 +29,14 @@ import org.opengis.filter.Filter
   * @tparam F wrapper around a simple feature - used for caching write calculations
   * @tparam WriteResult feature writers will transform simple features into these
   */
-trait GeoMesaFeatureIndex[DS <: GeoMesaDataStore[DS, F, WriteResult], F <: WrappedFeature, WriteResult] {
+trait GeoMesaFeatureIndex[DS <: GeoMesaDataStore[DS, F, WriteResult], F <: WrappedFeature, WriteResult]
+    extends LazyLogging {
 
   type TypedFilterStrategy = FilterStrategy[DS, F, WriteResult]
 
   lazy val identifier: String = s"$name:$version"
 
-  lazy val tableNameKey: String = s"table.$name.v$version"
+  lazy private val tableNameKey: String = s"table.$name.v$version"
```

Comment:
Hmmm... I wonder if this should be private or public.

Reply:
i made it private as the partition logic means that there won't always be a value under this exact key, so you should use the getTableNames method instead